### PR TITLE
Submission actions

### PR DESCRIPTION
**Pull request by phlax**
add submission actions and remove actions from the submission types field

also will require a migration to calc/set the actions and update the old type fields

### CODE REVIEW

File: pootle/apps/pootle_statistics/models.py [hunk 1]
```diff
@@ -25,6 +27,17 @@
 SIMILARITY_THRESHOLD = 0.5
 
 
+class SubmissionActions(Enum):
+    EDIT = 1
+    REVIEW = 2
+    REVERT = 3
+    SUGG_ADD = 4
+    SUGG_ACCEPT = 5
+    SUGG_REJECT = 6
```
Comment (unho) [inline pootle/apps/pootle_statistics/models.py]: I suggest name all the actions like `VERB_WHAT`, for example `REJECT_SUGGESTION` or `EDIT_TRANSLATION`.

File: pootle/apps/pootle_statistics/models.py [hunk 2]
```diff
@@ -25,6 +27,17 @@
 SIMILARITY_THRESHOLD = 0.5
 
 
+class SubmissionActions(Enum):
+    EDIT = 1
+    REVIEW = 2
```
Comment (unho) [inline pootle/apps/pootle_statistics/models.py]: Review what?

File: pootle/apps/pootle_statistics/models.py [hunk 3]
```diff
@@ -25,6 +27,17 @@
...
+    REVERT = 3
```
Comment (unho) [inline pootle/apps/pootle_statistics/models.py]: Revert what?

**Comment by phlax** [comment]
@unho the "what" is expressed in the `field` field

**Comment by unho** [comment]
Fair point re "what".

**Comment by phlax** [comment]
this is no longer required

**Pull request closed without merging**